src/Config.hpp

```diff
@@ -121,7 +121,7 @@ class Config : NonCopyable
   std::string m_secondary_config_path;
 
   std::string m_base_dir = "";
-  std::string m_cache_dir = fmt::format("{}/.ccache", get_home_directory());
+  std::string m_cache_dir = "";
```

Comment:
If `$CCACHE_CONFIGPATH` is set then `config.set_cache_dir` will never be called so `m_cache_dir` will be left unset. On the other hand it would be OK to remove the interpretation of `CCACHE_CONFIGPATH` since it's no longer used in the test suite.

Reply:
Oh, it was there only for the test suite?  I’ll remove support for it then.